html/cross-origin-opener-policy/resources/common.js

```diff
@@ -12,12 +12,11 @@ function url_test(t, url, channelName, hasOpener, openerDOMAccess) {
     if (openerDOMAccess !== undefined) {
       assert_equals(payload.openerDOMAccess, openerDOMAccess, 'openerDOMAccess');
     }
+    assert_equals(w.closed, !hasOpener, 'Openee window proxy closed');
   });
 
   const w = window.open(url, channelName);
 
-  // w will be closed by its postback iframe. When out of process,
-  // window.close() does not work.
```

Comment:
You want to grep for "postback frame" and adjust similar callsites/comments I think. There's 2 or 3 in total iirc.

Reply:
(Also check the cross-origin-embedder-policy directory.)